blogposts/2022/code-insights-ga-announcement.md

```diff
@@ -0,0 +1,94 @@
+
+What we've heard from organizations using Code Insights in Beta:
+
+- A greater number of meaningful codebase improvements get started _and_ finished.
+- Engineers are better recognized and rewarded for important progress that was previously hard to quantify.
+- Teams are more motivated to finish projects—visualizing their progress with Code Insights makes that progress more tangible and easier to communicate to leadership and other teams.
+
+## Why we had to build Code Insights
+
+Eng teams and leaders are a sophisticated bunch, but the current toolbox is failing them. There’s nothing out there with the visibility they need—because before you can even begin thinking about historical trends, you need a reliably exhaustive code search tool. So what they do today is try to source the information and visibility in a variety of hodgepodge, expensive ways that do not scale:
+
+- Asking a million people to provide status reports (and getting answers like the above)
+- Manually running brittle scripts that don't scale
+- Building workaround solutions in-house is limited to specific areas and only work if you remember to set them up at the very start of an initiative
+
+See? Lots of time, resources, and frustration, and the outcome just isn’t great.
+
+We started exploring Code Insights because our engineering teams faced the same problems. And it was an exciting extension of the Code Search functionality we’d already built: What if you could take anything you could search for and see how the result has changed over time? We know the truth is in the code, so the idea of unleashing this information for engineering teams and their leaders seemed very compelling.
+
+Then, as we built from concept to prototype to Beta to General Availability today, we were consistently surprised by how eager our customers were to start using it. Some of them would even snoop around our GitHub and changelog drafts to find out what features would arrive in each release before we shared the updates. It turns out we weren’t the only ones who wanted a tool to finally set KPIs on custom metrics based directly on the source of truth—the code—rather than manual spreadsheets or ad hoc updates.
+
+<figure>
+  <img src="https://storage.googleapis.com/sourcegraph-assets/blog/code-insights-ga-blogs/code-insights-dashboard.svg" alt="Code Insights dashboard" class="no-shadow">
+</figure>
+
+## Beyond code search
+
+Code Insights has huge momentum after an incredible year of growth at Sourcegraph. More than 1.275 million devs used Sourcegraph over the last 12 months, including many of the world's leading engineering teams, like 3/5 FAANG, Reddit, Uber, GE, PayPal, and Dropbox. Those teams first came to Sourcegraph for Code Search, which remains foundational to everything we do as we’ve expanded our product with Code Insights and [Batch Changes](https://about.sourcegraph.com/blog/introducing-batch-changes/).
```

Comment:
@amieroth Noting that this line in particular needs review/revision before the launch

Reply:
@rebeccadee thoughts on changing the first sentence to "Code Insights is a core part of our growth this year. More than 1.275. million..." I think the rest works fine, but LMK if there's anything in particular you feel is off given the more neutral tone we're looking for.

Reply:
```suggestion
Code Insights is a core part of our growth this year. More than 1.275 million devs used Sourcegraph over the last 12 months, including many of the world's leading engineering teams, like 3/5 FAANG, Reddit, Uber, GE, PayPal, and Dropbox. Those teams first came to Sourcegraph for Code Search, which remains foundational to everything we do as we’ve expanded our product with Code Insights and [Batch Changes](https://about.sourcegraph.com/blog/introducing-batch-changes/).
```